tests/functional/oauth_reset_password.js

```diff
@@ -49,211 +59,112 @@ define([
-        .end()
-
-        .findByCssSelector('a[href="/reset_password"]')
-          .click()
-        .end()
-
-        .findById('fxa-reset-password-header')
-        .end()
+        .then(testElementExists('#fxa-signin-header .service'))
+        .then(click('a[href="/reset_password"]'))
 
+        .then(testElementExists('#fxa-reset-password-header'))
         .then(fillOutResetPassword(email))
 
-        .findById('fxa-confirm-reset-password-header')
-        .end()
+        .then(testElementExists('#fxa-confirm-reset-password-header'))
 
-        .then(function () {
-          return FunctionalHelpers.openVerificationLinkInNewTab(
-                      self, email, 0);
-        })
+        .then(openVerificationLinkInNewTab(this, email, 0))
 
         // Complete the reset password in the new tab
         .switchToWindow('newwindow')
-
-        .findById('fxa-complete-reset-password-header')
-        .end()
-
-        .then(function () {
-          return FunctionalHelpers.fillOutCompleteResetPassword(self, PASSWORD, PASSWORD);
-        })
+        .then(testElementExists('#fxa-complete-reset-password-header'))
+        .then(fillOutCompleteResetPassword(this, PASSWORD, PASSWORD))
 
         // this tab's success is seeing the reset password complete header.
-        .findByCssSelector('#fxa-reset-password-complete-header')
-        .end()
-
-        .findByCssSelector('.account-ready-service')
-        .getVisibleText()
-        .then(function (text) {
-          // user sees the name of the rp,
-          // but cannot redirect
-          assert.isTrue(/123done/i.test(text));
-        })
-        .end()
-
+        .then(testElementExists('#fxa-reset-password-complete-header'))
+        // user sees the name of the rp, but cannot redirect
+        .then(testElementTextInclude('.account-ready-service', '123done'))
         .then(closeCurrentWindow())
 
         // the original tab should automatically sign in
-        .findByCssSelector('#loggedin')
-        .end();
+        .then(testElementExists('#loggedin'));
     },
 
     'reset password, verify same browser with original tab closed': function () {
-      var self = this;
-
-      return FunctionalHelpers.openFxaFromRp(self, 'signin')
-
-        .findByCssSelector('.reset-password')
-          .click()
-        .end()
+      return this.remote
+        .then(openFxaFromRp(this, 'signin'))
+        .then(click('.reset-password'))
 
         .then(fillOutResetPassword(email))
-
-        .findByCssSelector('#fxa-confirm-reset-password-header')
-        .end()
+        .then(testElementExists('#fxa-confirm-reset-password-header'))
 
         // user browses to another site.
-        .then(FunctionalHelpers.openExternalSite(self))
-
-        .then(function () {
-          return FunctionalHelpers.openVerificationLinkInNewTab(
-                      self, email, 0);
-        })
+        .then(FunctionalHelpers.openExternalSite(this))
```

Comment:
This little chap appears to be the only one who didn't get his name aliased at the top. Worth doing so he can be invoked without the `FunctionalHelpers.` prefix?